core/client/fs/src/main/java/alluxio/client/meta/MetaMasterConfigClient.java

```diff
@@ -75,4 +76,13 @@ void setPathConfiguration(AlluxioURI path, Map<PropertyKey, String> properties)
    * @param path the path
    */
   void removePathConfiguration(AlluxioURI path) throws IOException;
+
+  /**
+   * Update properties.
+   *
+   * @param propertiesMap the properties map to be updated
+   * @return the update properties status map
+   */
+  Map<String, UpdatePropertyPStatus> updateConfiguration(
+      Map<String, String> propertiesMap) throws IOException;
```

Comment:
How about the following?

```suggestion
  Map<PropertyKey, Boolean> updateConfiguration(
      Map<PropertyKey, String> propertiesMap) throws IOException;
```

1. using `PropertyKey` to indicate key as other methods in this interface to keep consistent, and more type-safe
2. let's not expose Proto-generated type to upper layer, to avoid using `UpdatePropertyPStatus`. It is very hard to read and understand the type when it is a proto

Reply:
For 2, it make sense.

For 1, It could not easy to use `PropertyKey` here, because, if user update some bad key which is not exist in PropertyKey, so we cannot get a PropertyKey at all, but we should return the result and tell the user failed to update that key.

Reply:
In that case, we can return to the caller of this client earlier when failed to convert a input string to `PropertyKey`, without really calling RPC. does that work?

Reply:
I have a case, if an old version shell client `Alluxio-V0` and a new version master `Alluxio-V1`, the V0 to V1 contains a new PropertyKey added, so we cannot update the new added PropertyKey through the  `Alluxio-V0` shell client. That make things getting hard to maintain the client and master to same version all the time.